index.html

```diff
@@ -1,163 +1,273 @@
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-            </ul>
             <div>
-                <button type="button">See Project</button>
+              <fieldset>
+                <textarea
+                  name="comment"
+                  placeholder="Write your message here"
+                  maxlength="500"
+                  required
+                ></textarea>
+              </fieldset>
             </div>
-        </div>
-    </section>
-    <section class="about">
-        <div class="about-content">
-            <div class="container">
-                <h2>About myself</h2>
-                <p>Hello I'm a software developer! I can help you build a product,  
-                feature or website. Look through some of my work and experience!
-                If you like what you see and have a project you need coded, don't hesitate 
-                to contact me. 
-                </p>
-                <span>LET'S CONNECT</span>
-                <ul class="social-media">
-                    <li><a href="#"><img src="./img/twitter.png" alt="twitter icon"></a></li>
-                    <li><a href="https://www.linkedin.com/in/jiachun-lily-xiang-1b197213a/"><img src="./img/linkedin.png" alt="linkedin icon"></a></li>
-                    <li><a href="#"><img src="./img/M.png" alt="M icon"></a></li>
-                    <li><a href="https://github.com/mpa-mxiang"><img src="./img/git.png" alt="github icon"></a></li>
-                    <li><a href="#"><img src="./img/hand.png" alt="hand icon"></a></li>
-                </ul>
-                <a><button type="button">Get my resume</button></a>  
 
-                <div class="languages">
-                    <h3>Languages</h3>
-                    <i class="fa-solid fa-chevron-down fa-xl"></i> 
-                </div>
-                <div class="language">
-                    <img class="language-img" src="./img/js.png" alt="JavaScript image">
-                    <p>JavaScript</p>
-                </div>
-                <div class="language">
-                    <img class="language-img" src="./img/html.jpg" alt="JavaScript image">
-                    <p>HTML</p>
-                </div>
-                <div class="language">
-                    <img class="language-img" src="./img/css.jpg" alt="JavaScript image">
-                    <p>CSS</p>
-                </div>
-                <div class="languages">
-                    <h3>Frameworks</h3>
-                    <i class="fa-solid fa-chevron-right fa-xl"></i> 
-                </div>
-                <div class="languages">
-                    <h3>Skills</h3>
-                    <i class="fa-solid fa-chevron-right fa-xl"></i> 
-                </div>
-            </div>
-            
+          </form>
+        </div>
+          <div class="btn-container">
+            <button class="button" type="submit">Get in touch</button>
         </div>
```

Comment:
Currently, when I fill the fields up with data and hit the submit button, the form does not submit and I believe it is because of this wrapping `div`. So I suggest you remove it so the form gets submitted once the button is pressed.

Furthermore, the interactivity (hover and active state) with the button has not been implemented according to the requirements. Having this implemented on your buttons will make them lively to the user as they can interact and understand what the button can do or is doing because of each state, please kindly implement this.

Reply:
It is not active working because the button was put outside of the `<form>`